### DISCUSSION TITLE
Is it normal for the app to take a while to sync? It's stuck at 8%

### DISCUSSION THREAD

**Discussion by danmanrana** [ACINQ/eclair#2189] [Q&A] [answered]
I'm using the latest version on Android. Creating a new wallet. A forum comment suggests changing the server but I don't see any option to.

**Comment by t-bast** [comment, marked as answer]
I think you're using eclair-mobile, not eclair. You should ask your future question there: https://github.com/ACINQ/eclair-mobile
What is slow here is syncing the lightning network graph, not the blockchain, so changing the server won't have any impact.
Unfortunately there is a lot of data to sync for a new wallet, once that's done a background job will regularly sync the latest updates.
An alternative would be to use [Phoenix](https://phoenix.acinq.co/) which doesn't need to sync the graph at all.
